SAMPLE_FORMAT.md

```diff
@@ -348,33 +348,39 @@ public static void exampleSnippet(String projectId, String filePath) {
     // Snippet content ...
```

Comment:
nit: maybe update this to: `public static String exampleSnippet(...` to show a non-null return type?